docs/lib/datastore/getting-started.md

```diff
@@ -9,44 +9,161 @@ Amplify DataStore provides a programming model for leveraging shared and distrib
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/10_preReq.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/10_preReq.md"></inline-fragment>
+
+## Install Amplify Libraries
+
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/20_installLib.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/20_installLib.md"></inline-fragment>
+
+There are two options to integrate the Amplify build process with the project.
+
+### Option 1: Platform integration
+
+<inline-fragment platform="js" src="~/lib/datastore/fragments/js/getting-started/30_platformIntegration.md"></inline-fragment>
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/30_platformIntegration.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/30_platformIntegration.md"></inline-fragment>
+
+### Option 2: Use Amplify CLI
+
+Instead of using the platform integration, you can alternatively use the Amplify CLI on its own to accomplish the same thing that Amplify Tools is doing for you. This option is particularly useful for **existing projects** where Amplify is already configured.
+
+The base structure for a DataStore app is created by adding a new GraphQL API to your app.
+
+```console
+amplify add api
+```
+
+During the API configuration process select **GraphQL** as the API type and reply to the questions as follows. Make sure you respond **Yes, I want to make some additional changes** when prompted for **advanced settings** and turn on **conflict detection**. This setting is critical when syncing data to the cloud since the conflict resolution strategy is what allows local data to be reconciled with data from the cloud backend.
+
+```console
+? Please select from one of the below mentioned services:
+    `GraphQL`
+? Provide API name:
+    `BlogAppApi`
+? Choose the default authorization type for the API
+    `API key`
+? Enter a description for the API key:
+    `BlogAPIKey`
+? After how many days from now the API key should expire (1-365):
+    `365`
+? Do you want to configure advanced settings for the GraphQL API
+    `Yes, I want to make some additional changes.`
+? Configure additional auth types?
+    `No`
+? Configure conflict detection?
+    `Yes`
+? Select the default resolution strategy
+    `Auto Merge`
+? Do you have an annotated GraphQL schema?
+    `No`
+? Do you want a guided schema creation?
+    `No`
+? Provide a custom type name
+    `Post`
+```
+
+<amplify-callout warning>
+
+**Troubleshooting:** without the **conflict detection** configuration cloud sync will fail. In that case use `amplify update api` and choose **Enable DataStore for entire API** (this option will enable the conflict detection as described above).
```

Comment:
I would make the previous reference to this more obvious. Is strange that **Enable DataStore for entire API** is only on `api update` and not on `api add` command

Reply:
Yeah, I agree that that question being only present in the update flow is weird.

Can you elaborate on *"I would make the previous reference to this more obvious"*?